Automattic/WP-Job-Manager
# Add support for remote positions

Fixes #918 

#### Changes proposed in this Pull Request:
When Adding a new Job Listing, now exists the ability to consider that as a Remote Position
If its Tagged as a Remote Position the Google Job Search Schema will be updated accordingly
Search for Remote Positions is available via the Location field.

#### Testing instructions:
Install WordPress and WP Job Manager
To add a Remote Position just check the Remote Position checkbox when adding a new listing
To search for remote positions, type "remote" or "Remote" in the Location Field


#### Proposed changelog entry for your changes: Add Support for Remote Positions

A CSS update will be needed when is a Remote Position to get it with the same look as a "normal" position, but i want to improve this at the end.

Any feedback to improve this PR is very much appreciated. 

Screenshots here
<img width="1279" alt="screenshot 2018-11-21 at 14 37 59" src="https://user-images.githubusercontent.com/36240423/48848903-156e5200-ed9d-11e8-84aa-93462ea2f7b1.png">
<img width="943" alt="screenshot 2018-11-21 at 14 38 36" src="https://user-images.githubusercontent.com/36240423/48848904-156e5200-ed9d-11e8-874c-9b94b53f05ba.png">
<img width="933" alt="screenshot 2018-11-21 at 14 38 50" src="https://user-images.githubusercontent.com/36240423/48848905-156e5200-ed9d-11e8-88b2-7197c7ac4391.png">
